config/local.sh

```diff
@@ -16,3 +16,4 @@ export MATOMO_TRACKER_URL='disable'
 export MAX_FAILED_LOGIN_ATTEMPTS='5'
 export DELAY_BEFORE_NEXT_LOGIN_ATTEMPT='10'
 export TERMS_PRIVACY_DATE='2020-02-26'
+export CROWDIN='disable'
```

Comment:
```suggestion
export CROWDIN='disabled'
```
I know other var is "disable" (it was my mistake), but I see everywhere "disabled" for this situation.
Should we set it in the past tense ?